Integrations/Integration-Salesforce.yml

```diff
@@ -0,0 +1,174 @@
+configuration:
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Consumer Key
```

Comment:
shouldn't the secret and the key be an authentication type as they are bounded together?
usually we use camel case notation and not _ in params